src/SeleniumLibrary/__init__.py

```diff
@@ -223,6 +224,89 @@ class SeleniumLibrary(DynamicCore):
 
     See the `Add Location Strategy` keyword for more details.
 
+    = Browser and Window =
+
+    In this section the concept of Browsers or Webdrivers and its Windows
+    will be explained.
+
+    == Browser ==
+
+    In Selenium Library there are two ways to start a Browser and control it.
```

Comment:
Hmmm, I am hesitant about this chapter. Would it be better to explain something like this: 

_Each time the Open Browser or Create WebDriver keywords are called, it will create a new browser. Also Internally it creates a Selenium WebDriver_ [link here selenium docs would be nice] _instance. The new browser is, usually, independent from other browsers opened by the in the test, example they do not share cookies or example profile_

I would not explain the what keywords are preferred and about the keyword history, I think it confuses the reader. The above example is quite minimal and requires some more text.

Reply:
Did refactor this chapter.
More details on Webdriver and Selenium Stuff.